### PR TITLE
Move onnx-caffe2 into caffe2

### PR DESCRIPTION
**Pull request by yinghai**
Since we have moved `onnx-caffe2` into `caffe2`. Update the documentation here.

### CODE REVIEW

**Comment by yinghai** [comment]
@houseroad @dzhulgakov

File: tutorials/PytorchCaffe2SuperResolution.ipynb [hunk 1]
```diff
@@ -18,9 +18,9 @@
     "executing the model on mobile devices.\n",
     "\n",
     "For this tutorial, you will need to install [onnx](https://github.com/onnx/onnx),\n",
-    "[onnx-caffe2](https://github.com/onnx/onnx-caffe2) and [Caffe2](https://caffe2.ai/).\n",
-    "You can get binary builds of onnx and onnx-caffe2 with\n",
-    "``conda install -c ezyang onnx onnx-caffe2``.\n",
+    "and [Caffe2](https://caffe2.ai/).\n",
+    "You can get binary builds of onnx with\n",
+    "``conda install -c ezyang onnx``.\n",
```
Comment (houseroad) [inline tutorials/PytorchCaffe2SuperResolution.ipynb]: here, we should use conda-forge instead of ezyang's channel.

Comment (yinghai) [reply tutorials/PytorchCaffe2SuperResolution.ipynb]: Suggest code change?

Comment (houseroad) [reply tutorials/PytorchCaffe2SuperResolution.ipynb]: conda install -c ezyang onnx ==> conda install -c conda-forge onnx

A different issue, but it's good to solve it.

**Comment by yinghai** [comment]
Updated.

**Comment by dzhulgakov** [comment]
Thank you!

**Pull request merged**